config/locales/diaspora/en.yml

```diff
@@ -345,6 +345,10 @@ en:
     tutorials: "tutorials"
     tutorial: "tutorial"
     irc: "IRC"
+    closed_pod: "This pod is currently closed to new registrations. However, you can still join the diaspora* network by registering on %{wiki}. Because all pods are interconnected, you will have access to the same content there."
+    find_pods: "You can find pods at %{poduptime}."
```

Comment:
Maybe "You can use %{poduptime} to choose a pod."?

Reply:
@Flaburgan I think telling people that pod uptime can help them choose a pod (or something like that) is better than simply that they can find pods there. Alternatively, `"There’s a list of pods you can sign up to at %{poduptime}."`

Reply:
OK changed.